### PR TITLE
Tag enhanced metrics with datadog lambda library version

### PR DESCRIPTION
**Pull request by Czechh**
### What does this PR do?
Adds `datadog_lambda:v${version}` tag to enhanced metrics

### Motivation
This will allow us to aggregate the library versions used by our customers.

### Types of Changes

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [X] Misc (docs, refactoring, dependency upgrade, etc.)

### Check all that apply

- [ ] This PR's description is comprehensive
- [ ] This PR contains breaking changes that are documented in the description
- [ ] This PR introduces new APIs or parameters that are documented and unlikely to change in the foreseeable future
- [ ] This PR impacts documentation, and it has been updated (or a ticket has been logged)
- [X ] This PR's changes are covered by the automated tests
- [ ] This PR collects user input/sensitive content into Datadog
- [X] This PR passes the integration tests (ask a Datadog member to run the tests)

### CODE REVIEW

**Review by tianchu — APPROVED** [review]

File: src/metrics/enhanced-metrics.ts [hunk 1]
```diff
@@ -16,6 +17,10 @@ enum RuntimeTagValues {
   Node12 = "nodejs12.x",
 }
 
+export function getVersionTag(): string {
+  return `datadog_lambda:v${datadogLambdaVersion}`;
+}
```
Comment (hghotra) [inline src/metrics/enhanced-metrics.ts:22]: Small nit - maybe we should put this function in utils somewhere? Exporting a DD lambda version tag function from `enhanced-metrics.ts` seems a bit off.

Comment (Czechh) [reply src/metrics/enhanced-metrics.ts:22]: Great call out, I moved that to the parent folder 👍

Comment (Czechh) [reply src/metrics/enhanced-metrics.ts:22]: Oh I thought you meant importing the `datadogLambdaVersion` from trace constants. It was something I was thinking about. I think the export here is mainly for testing. But, this function is really not used anywhere else. `datadogLambdaVersion` is used in trace tags which are set as an object so it's not exactly the same

**Comment by Czechh** [comment]
@tianchu I've rebased my changes + updated the integration tests

**Review by tianchu — APPROVED** [review]

**Pull request merged**